stable/spring-cloud-data-flow/values.yaml

```diff
@@ -16,19 +16,45 @@ initContainers:
     tag: "1.30.1"
     imagePullPolicy: IfNotPresent
 
+nameOverride: ""
+fullnameOverride: ""
+
 server:
   image: springcloud/spring-cloud-dataflow-server
   version: 2.3.0.RELEASE
   imagePullPolicy: IfNotPresent
   platformName: default
   trustCerts: false
+  ingress:
+    enabled: false
+    annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    labels: {}
+    path: /
+    hosts:
+      - chart-example.local
+    ## Extra paths to prepend to every host configuration. This is useful when working with annotation based services.
+    extraPaths: []
+    # - path: /*
+    #   backend:
+    #     serviceName: ssl-redirect
+    #     servicePort: use-annotation
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
   service:
-    type: LoadBalancer
-    externalPort: 80
+    type: ClusterIP
```

Comment:
by default SCDF should be exposed via LoadBalancer and toggled otherwise via `set`

Reply:
Ingress is usually controller by Ingress resources and not Load Balancers. Load Balancers are costly and usually in production environments there is only a couple of ALB (or ELB) attached to the cluster. With ClusterIP (or NodePort) as default, it's up for the k8s administrator on how to expose the pods. LoadBalancer is nice for docker-for-desktop though. I'll update and make this configurable

Reply:
sure; depends on what your env is. lets keep this as LB and it can be overridden in envs where needed.

Reply:
Reverted.